presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/CachingJdbcClient.java

```diff
@@ -225,7 +225,6 @@ public JdbcOutputTableHandle beginInsertTable(ConnectorSession session, JdbcTabl
     public void finishInsertTable(ConnectorSession session, JdbcOutputTableHandle handle)
     {
         delegate.finishInsertTable(session, handle);
-        invalidateTableCaches(handle.getSchemaTableName());
```

Comment:
Drop the change. The insert may change the state of the table, including its obvervable metadata.

Reply:
Could you please share a simple example?

Reply:
Please extract this to separate pull request, so we can discuss this independently.